packages/react-storage/src/components/StorageBrowser/composables/ActionCancel.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { ButtonElement, IconElement } from '../context/elements';
+import { CLASS_BASE } from '../views/constants';
+
+export interface ActionCancelProps {
+  onCancel?: () => void;
+  ariaLabel?: string;
+  isDisabled?: boolean;
+  text?: string;
+}
+
+export const ActionCancel = ({
+  onCancel,
+  ariaLabel,
+  isDisabled,
+  text,
```

Comment:
This should be `label` to align with `ActionStart` right?

Reply:
Won't it be confusing with ariaLabel?

Reply:
Reverted to label

Reply:
We're using `label` for the `ActionStart` button, tbh not really sure why we have `ariaLabel` here, control props aren't meant to be that low level